pkg/images/deps/provider.go

```diff
@@ -38,11 +41,16 @@ func NewRootProvider(
 		sentry.NewLogHookFromHub(sentryHub),
 	)
 
+	// We do not have a chance to close this yet :(
+	// But it is not harmful not to close this.
+	vipsDaemon := vipsutil.OpenDaemon(runtime.NumCPU())
```

Comment:
How about we open daemon it in the controller `./cmd/authgear/images/server/server.go` and pass it to the RootProvider constructor?